x-pack/plugins/fleet/public/components/agent_enrollment_flyout/kubernetes_instructions.tsx

```diff
@@ -114,8 +111,13 @@ export const KubernetesInstructions: React.FunctionComponent<Props> = ({
     </EuiCopy>
   );
 
+  const searchParams = new URLSearchParams({
+    ...(fleetServerHost && { fleetServer: fleetServerHost }),
+    ...(enrollmentAPIKey && { enrolToken: enrollmentAPIKey }),
+  });
+
   const downloadLink = core.http.basePath.prepend(
-    `${agentPolicyRouteService.getK8sFullDownloadPath()}?fleetServer=${fleetServer}&enrolToken=${enrollmentAPIKey}`
+    `${agentPolicyRouteService.getK8sFullDownloadPath()}${searchParams.toString()}`
```

Comment:
How heavy of a lift would it be to add a test case to assert on the expected value showing up in the generated instructions? I know there's not a test file here right now, but maybe worth a quick try and possibly an issue if it's going to be too involved?

Reply:
Just a unit test I think would be appropriate.

Reply:
I opted for a cypress test in the end: https://github.com/elastic/kibana/pull/165127/commits/48fd6d9097f86309d9cac42ac74b8762bcd5ab67